setup.py

```diff
@@ -1,18 +1,17 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 import sys
 
 CLASSIFIERS = [
     "Development Status :: 4 - Beta",
     "Natural Language :: English",
     "License :: OSI Approved :: MIT",
     "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
```

Comment:
What's driving this change, specifically in this PR?

We likely need it regardless for 3.0, especially if we will completely type-hint, but let's document the reason here.

Reply:
I'll add back python 3.6 as I'm not sure we need to delete this for this PR